tubemq-core/src/main/java/org/apache/tubemq/corebase/utils/CompressionType.java

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Compression processing and compression enumeration need to be separated: at present, there is no problem when there is only one compression method, but subsequent compression methods are added, some methods need to be initialized, and some require other processing, the enumeration class will become unstable, and need to be adjusted frequently. 

So enumerating compression methods and specific compression classification will be better to solve this problem